lib/msal-common/src/authority/AuthorityFactory.ts

```diff
@@ -10,6 +10,7 @@ import { StringUtils } from "../utils/StringUtils";
 import { ClientAuthError } from "../error/ClientAuthError";
 import { ICacheManager } from "../cache/interface/ICacheManager";
 import { AuthorityOptions } from "./AuthorityOptions";
+import { Constants } from "..";
```

Comment:
This will create a circular dependency. Always import from the file, not the index

Reply:
oops. Not sure how that got in there.